.travis.yml

```diff
@@ -70,4 +70,5 @@ before_script: >
     . $HOME/staging/share/GNUstep/Makefiles/GNUstep.sh;
 script: >
     ./configure $BASE_ABI || (cat config.log && false);
-    make && make install && make check || (cat Tests/tests.log && false);
+    make && make install
+    # make check || (cat Tests/tests.log && false);
```

Comment:
Why is this included in this pull request? Did you have issues with the base tests?
At least we should do this separately from this PR.

Reply:
Arghhh... I used git commit -am ... instead of -m.   I'll correct it.

Reply:
Off topic: I don't think this change originated on my branch.  I had done a rebase from master that may have picked this up.   Also, I don't think we use the travis.yml stuff any more since github made a change moving away from it.

Reply:
`3627b22481 (fredkiefer             2019-12-06 17:32:25 +0100 72)     ./configure $BASE_ABI || (cat config.log && false);
94cc250d2d (Gregory John Casamento 2021-07-15 11:36:08 -0400 73)     make && make install
94cc250d2d (Gregory John Casamento 2021-07-15 11:36:08 -0400 74)     # make check || (cat Tests/tests.log && false);`

Strange, I don't recall making this change.  Nevertheless, I am obliterating it.

Reply:
At least it is not on the master branch.

Reply:
I have removed the changes.